index.html

```diff
@@ -18,13 +17,17 @@
             <header class="navbar">
                 <a href="#">RODCHAK</a>
 
-                <nav><a href="#">
-                        <span class="material-symbols-outlined mobile">menu</span></a>
-                    <ul class="menu_bar desktop">
-                        <li> <a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li> <a href="#">Contact</a></li>
+                <nav>
+                    <ul class="menu_bar">
+                        <li class="nav-item"><a href="#Works_Section" class="nav-link">Portfolio</a></li>
+                        <li class="nav-item"><a href="#About_Myself" class="nav-link">About</a></li>
+                        <li class="nav-item"><a href="#Contact_form" class="nav-link">Contact</a></li>
                     </ul>
+                    <div class="hamburger">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
```

Comment:
- [ ] Also make sure to add the correct spacing around the `Hamburger`(top ** right) button to stick to the Figma design. For the `Top Spacing` I suggest you add to the whole top of the **Header**  then the right of the `Hamburger` button to stick to the Figma design.

| ### Figma DEsign | Yout Design |
| ------------------------- | ------------------------ |
| ![Screenshot from 2022-07-19 21-21-28](https://user-images.githubusercontent.com/83042742/179831674-93577cfc-1720-4756-839b-64fbe0ee5c0f.png) | ![Screenshot from 2022-07-19 21-22-50](https://user-images.githubusercontent.com/83042742/179831891-1200ceb7-db3c-4346-81f6-3a0616fcaf6e.png) |